nisystemlink/clients/assetmanagement/models/_query_location.py

```diff
@@ -0,0 +1,131 @@
+from enum import Enum
+from typing import List, Optional
+
+from nisystemlink.clients.core._api_error import ApiError
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+
+
+class ResponseFormat(Enum):
+    """Gets or sets the return type. Valid options are "JSON" and "CSV"."""
+
+    JSON = "JSON"
+
+    CSV = "CSV"
+
+
+class Destination(Enum):
+    """Gets or sets the destination of the request."""
+
+    INLINE = "INLINE"
+    """(default) Returns the list of resources as the body of the response."""
+
+    DOWNLOAD = "DOWNLOAD"
+    """Returns the list of resources as the body of the response and it should be downloaded as a file."""
+
+    FILE_SERVICE = "FILE_SERVICE"
+    """Sends the list of resources to the file ingestion service and returns the ID of the file in a JSON object."""
+
+
+class SystemConnection(Enum):
```

Comment:
You've duplicated SystemConnection and AssetPresence. These are the same for all the APIs.